src/js/control-bar/text-track-controls/chapters-button.js

```diff
@@ -124,7 +98,8 @@ class ChaptersButton extends TextTrackButton {
 
         const mi = new ChaptersTrackMenuItem(this.player_, {
           cue,
-          track: chaptersTrack
+          track: chaptersTrack,
+          selectable: true
```

Comment:
Exact implementation of https://github.com/videojs/video.js/blob/master/src/js/control-bar/text-track-controls/text-track-button.js#L49.